components/activity_and_insights/activity_and_insights_link/insights_tour_tip/insights_tour_tip.tsx

```diff
@@ -0,0 +1,131 @@
+import FormattedMarkdownMessage from 'components/formatted_markdown_message.jsx';
+import {getConfig} from 'mattermost-redux/selectors/entities/general';
+import {showNextSteps} from 'components/next_steps_view/steps';
+import {GlobalState} from 'types/store';
+import {GlobalState as EntitiesGlobalState} from 'mattermost-redux/types/store';
+import {isFirstAdmin} from 'mattermost-redux/selectors/entities/users';
+import {getBool, getUseCaseOnboarding} from 'mattermost-redux/selectors/entities/preferences';
+import {OnboardingTaskCategory, OnboardingTaskList} from 'components/onboarding_tasks';
+
+const title = (
+    <FormattedMessage
+        id='activityAndInsights.tutorialTip.title'
+        defaultMessage='Introducing: Insights'
+    />
+);
+
+const screen = (
+    <>
+        <FormattedMarkdownMessage
+            id='activityAndInsights.tutorialTip.description'
+            defaultMessage='Check out the new Insights feature added to your workspace. See what content is most active, and learn how you and your teammates are using your workspace.'
+        />
+        <img
+            src={insightsPreview}
+            className='insights-img'
+        />
+    </>
+
+);
+
+const prevBtn = (
+    <FormattedMessage
+        id={'activityAndInsights.tutorial_tip.notNow'}
+        defaultMessage={'Not now'}
+    />
+);
+
+const nextBtn = (
+    <FormattedMessage
+        id={'activityAndInsights.tutorial_tip.viewInsights'}
+        defaultMessage={'View insights'}
+    />
+);
+
+const InsightsTourTip = () => {
+    const dispatch = useDispatch();
+    const showTip = useSelector(showInsightsPulsatingDot);
+    const config = useSelector(getConfig);
+    const nextSteps = useSelector((state: GlobalState) => showNextSteps(state));
+    const showNextStepsEphemeral = useSelector((state: GlobalState) => state.views.nextSteps.show);
+    const firstAdmin = useSelector(isFirstAdmin);
+    const useCaseOnboarding = useSelector(getUseCaseOnboarding);
+    const showTaskList = useSelector((state: EntitiesGlobalState) => getBool(state, OnboardingTaskCategory, OnboardingTaskList.ONBOARDING_TASK_LIST_SHOW));
+
+    const [tipOpened, setTipOpened] = useState(showTip);
+
+    const handleDismiss = useCallback((e: React.MouseEvent) => {
+        e.stopPropagation();
+        dispatch(setInsightsInitialisationState({[Preferences.INSIGHTS_VIEWED]: true}));
+        setTipOpened(false);
+    }, []);
+
+    const handleNext = useCallback(() => {
+        dispatch(setInsightsInitialisationState({[Preferences.INSIGHTS_VIEWED]: true}));
+        setTipOpened(false);
+    }, []);
+
+    const handleOpen = useCallback((e: React.MouseEvent) => {
+        e.stopPropagation();
+        setTipOpened(true);
+    }, []);
+
+    const isOnboardingOngoing = useCallback(() => {
+        if ((config.EnableOnboardingFlow === 'true' && nextSteps && !showNextStepsEphemeral && !(useCaseOnboarding && firstAdmin)) || (firstAdmin && showTaskList) || showNextStepsEphemeral) {
```

Comment:
@pvev as another outcome of the [show the onboarding task list to all users PR] (https://github.com/mattermost/mattermost-webapp/pull/10187), the `(firstAdmin && showTaskList)` condition will need to be changed to `showTaskList` once that PR is is merged, right?

I think the whole `!(useCaseOnboarding && firstAdmin)` condition can be removed. If use case onboarding is false, `showTaskList` will not be true, because it is code in the onboarding that sets the state showTaskList is based on.

Reply:
@neallred is right. The condition must change cause all the user will have the onboarding task list.

Reply:
I've updated this to match your new PR I think @pvev. Let me know if the `if statement` is correct

Reply:
Ok that's great! Thanks